### DISCUSSION TITLE
13 歲少年攻破〈俄羅斯方塊〉

### DISCUSSION THREAD

**Discussion by alvin119** [1101-GS4719-C/Python-Community-GS4719-C#1832] [General]
感覺俄羅斯方塊在近幾年快被玩壞了。
自從Cheez發明叫作rolling的技術後，以前的紀錄就不斷被打破，
雖然突破極限是好事，但我覺得這也導致俄羅斯方塊比賽變得只是在比誰不失誤，
看幾十分鐘還是會覺得有些無聊。
資料來源:https://technews.tw/2024/01/04/blue-scuti-beat-tetris/

**Comment by woohsin** [comment]
自從看了啾啾鞋的tetris系列之後就有稍微關注一下，這種一直找到新玩法打破目前記錄其實是好事吧，如果真的不行了代表這個遊戲已經到盡頭了(確實被玩家們玩也到盡頭了)
但是就算大家已經知道他的終點，他還是一款經典的遊戲，反正我也不可能學會rolling :)

**Comment by woohsin** [comment]
最近回去玩了，最大的意見就是tetris.io的音樂超大聲，雖然選曲都不錯，但是真的很吵。

**Comment by woohsin** [comment]
更)
還是學不會踢轉，但我把前幾年的比賽看了好多次。

**Comment by thomasxshen** [comment]
這種技術允許玩家在極短的時間內連續快速地放置方塊，從而破解了之前認為不可能突破的速度限制。

這種創新不僅打破了許多舊有紀錄，也推動了俄羅斯方塊社群對遊戲技巧和極限的重新評估。技術的進步經常會帶來這樣的結果，它促進了遊戲的發展，同時也挑戰了玩家的技能，使得他們必須不斷學習和適應新的玩法以保持競爭力。

然而這也可能使得比賽看起來更加單調，因為玩家可能過於依賴特定技術，減少了戰略的多樣性。這就要求組織者和社群考慮如何平衡技術創新與比賽的觀賞性和公平性，以保持遊戲的長期吸引力和可持續發展。

**Comment by wanka2030** [comment]
每項專業都有每個人努力的專研，不容許拿別人的專業挑戰自身的興趣。曾經也很常在fb上面遊玩俄羅斯方塊，也以為自己是金字塔頂端上面的1%，但長大後發現自己的世界外有許多高手、專業人士的存在，所以當一個人將自己的興趣變成專業，不得不讓人敬佩！

**Comment by yyujimmy** [comment]
俄羅斯方塊的魅力在於它的速度和技巧，但如果比賽過於注重於避免失誤，可能會讓比賽變得單調乏味。這也提醒我們，在追求極限的同時，還應該注重遊戲的樂趣和多樣性。

或許可以考慮一些改變或調整，讓比賽更具有挑戰性和刺激性，同時保留遊戲的原始樂趣。例如，引入新的比賽規則或模式，或者加入一些讓比賽更加精彩的元素。

無論如何，俄羅斯方塊作為一款經典的遊戲，一定會持續吸引著無數玩家，並且不斷演化和創新。

**Comment by Piyenparty** [comment]
現在感覺都是在比誰比較不會失誤，比賽變得單調，削弱戰術的多樣性與觀賞性。或許這提醒主辦方與社群需共同思考，如何在鼓勵技術創新的同時，兼顧比賽的公平性與娛樂性，確保遊戲能長期保持吸引力，不然再這樣下去接觸俄羅斯方塊的人口只會越來越少，以致最後沒落。
